assets/_scss/extends/_card.scss

```diff
@@ -0,0 +1,5 @@
+%card {
+  background-color: $color-white;
+  box-shadow: 0 -1px 0 lighten($color-gray-light, 25), 0 0 2px rgba(0, 0, 0, 0.05), 0 2px 4px rgba(0, 0, 0, 0.09);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.05)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `rgba(0, 0, 0, 0.09)` should only be used in variable declarations; they should be referred to via variable everywhere else.